src/code/z_view.c

```diff
@@ -511,7 +523,10 @@ s32 View_ApplyPerspectiveToOverlay(View* view) {
         view->eye.z += 1.0f;
     }
 
+#if OOT_DEBUG
     View_ErrorCheckEyePosition(view->eye.x, view->eye.y, view->eye.z);
+#endif
```

Comment:
with how often it comes up in the file, I kinda think `VIEW_ERROR_CHECK_EYE_POS(view)` is worth it to remove a bunch of #if 's. It can be a macro local to the file

Reply:
oh i didnt realize there was an external use. neat